Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -261,6 +279,10 @@ export interface QTestArguments extends Transformer.RunnerArguments {
     vstestSettingsFile?: File;
     /** Optionally override to increase the weight of test pips that require more machine resources */
     weight?: number;
+    /** Describes the type of coverage that QTest should employ. */
+    qCodeCoverageEnumType?: QCodeCoverageEnumType;

```

Comment:
>QCodeCoverageEnumType [](start = 28, length = 21)

Please don't add new Enums. It is better and cleaner to use union of string literal types.
```
qCodeCoverageEnumType?: "DynamicCodeCov" | "None";
```

This in the language has the same semantics as the enum it will check for illegal values.

Reply:
Fixed.